metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -33,39 +33,35 @@ class BatchDecorator(StepDecorator):
 
     Parameters
     ----------
-    cpu : int
-        Number of CPUs required for this step. Defaults to 1. If `@resources` is
+    cpu : int, default: 1
+        Number of CPUs required for this step. If `@resources` is
         also present, the maximum value from all decorators is used.
-    gpu : int
-        Number of GPUs required for this step. Defaults to 0. If `@resources` is
+    gpu : int, default: 0
+        Number of GPUs required for this step. If `@resources` is
         also present, the maximum value from all decorators is used.
-    memory : int
-        Memory size (in MB) required for this step. Defaults to 4096 (4GB). If
+    memory : int, default: 4096
+        Memory size (in MB) required for this step. If
         `@resources` is also present, the maximum value from all decorators is
         used.
-    image : string
+    image : str, optional, default: None
```

Comment:
the default isn't technically None here

Reply:
good catch. I'll change to default: BATCH_CONTAINER_IMAGE (I went by the default below but this is in the code for some reason.

Reply:
I actually left it as optional and added an explanation.